api/bonsai_api/cli/cli.py

```diff
@@ -154,25 +155,40 @@ def index(_ctx: click.Context):  # pylint: disable=unused-argument
 @cli.command()
 @click.pass_obj
 @click.option("-i", "--sample-id", required=True, help="Sample id")
+@click.option("-e", "--export-cnf", type=click.Path(), help="Optional LIMS export configuration.")
+@click.option("-f", "--format", "output_format", type=click.Choice(["csv", "tsv"]), help="Optional LIMS export configuration.")
 @click.argument("output", type=click.File("w"), default="-")
 def export(
-    _ctx: click.Context, sample_id: str, output: TextIOWrapper
+    _ctx: click.Context, sample_id: str, export_cnf: pathlib.Path | None, output_format: Literal["tsv", "csv"], output: TextIOWrapper,
```

Comment:
Surely this shouldn't be able to take None? Otherwise it shouldn't be run?

Reply:
If no export config is provided the CLI should default to the config packaged with Bonsai.